settings/Core.setting.php

```diff
@@ -42,17 +42,19 @@
     'group' => 'core',
     'name' => 'contact_view_options',
     'type' => 'String',
+    'quick_form_type' => 'CheckBoxes',
     'html_type' => 'checkboxes',
     'pseudoconstant' => array(
       'optionGroupName' => 'contact_view_options',
     ),
     'default' => '123456789101113',
     'add' => '4.1',
-    'title' => 'Viewing Contacts',
+    'title' => ts('Viewing Contacts'),
     'is_domain' => '1',
     'is_contact' => 0,
-    'description' => NULL,
+    'description' => ts("Select the tabs that should be displayed when viewing a contact record. EXAMPLE: If your organization does not keep track of 'Relationships', then un-check this option to simplify the screen display. Tabs for Contributions, Pledges, Memberships, Events, Grants and Cases are also hidden if the corresponding component is not enabled. Go to Administer > System Settings > Enable Components to modify the components which are available for your site."),
```

Comment:
ideally we also use this for the tpl help text rather than duplicating - I haven't looked at that as yet